qiskit/__init__.py

```diff
@@ -122,6 +131,15 @@ def __getattr__(self, attr):
                 from qiskit.providers import ibmq
 
                 self.ibmq = ibmq.IBMQ
+                warnings.warn(
+                    "The qiskit.IBMQ entrypoint and the qiskit-ibmq-provider package ("
+                    "accessible from 'qiskit.providers.ibmq`) are deprecated and will be removed "
+                    "in a future release. Instead you should use the qiskit-ibm-provider package "
+                    "which is accesible from 'qiskit_ibm_provider'. Just replace 'qiskit.IBMQ' "
+                    "with 'qiskit_ibm_provider.IBMProvider'",
```

Comment:
This part has an additional sentence "Just replace 'qiskit.IBMQ' with 'qiskit_ibm_provider.IBMProvider'" Is this intended?